applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_check_utilities.cpp

```diff
@@ -16,6 +16,27 @@
 
 using namespace Kratos;
 
+namespace
+{
+
+Geometry<Node> CreatLineGeometryWithoutVariables() {
+    PointerVector<Node> nodes;
+    nodes.push_back(make_intrusive<Node>(0, 0.0, 0.0, 0.0));
+    nodes.push_back(make_intrusive<Node>(1, 1.0, 0.0, 0.0));
+    return Line2D2<Node>(nodes);
+}
+
+Geometry<Node> CreatLineGeometryWithVariables() {
+    Model model;
+    auto& r_model_part = model.CreateModelPart("ModelPart", 1);
+    r_model_part.AddNodalSolutionStepVariable(WATER_PRESSURE);
+    PointerVector<Node> nodes;
+    nodes.push_back(r_model_part.CreateNewNode(0, 0.0, 0.0, 0.0));
+    nodes.push_back(r_model_part.CreateNewNode(1, 1.0, 1.0, 1.0));
```

Comment:
Nitpicking: when creating a 2D geometry (like here), it seems a bit awkward to have a non-zero Z coordinate.
```suggestion
    nodes.push_back(r_model_part.CreateNewNode(1, 1.0, 1.0, 0.0));
```

Reply:
fixed.